modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/SkinMemoryLeakTest.java

```diff
@@ -170,8 +170,8 @@ public static Collection<Object[]> data() {
                 MenuButton.class,
                 Pagination.class,
                 PasswordField.class,
-                ScrollBar.class,
-                ScrollPane.class,
+                //ScrollBar.class,
```

Comment:
We can remove these lines instead of commenting them out.

Reply:
@aghaisas : 
will clean up this area once the last PR is integrated, otherwise we'll have constant merge conflicts here (every skin memory leak PR touches this)